content/en/docs/reference/command-line-tools-reference/feature-gates-removed.md

```diff
@@ -47,6 +47,9 @@ In the following table:
 | `BoundServiceAccountTokenVolume` | `false` | Alpha | 1.13 | 1.20 |
 | `BoundServiceAccountTokenVolume` | `true` | Beta | 1.21 | 1.21 |
 | `BoundServiceAccountTokenVolume` | `true` | GA | 1.22 | 1.23 |
+| `CPUManager` | `false` | Alpha | 1.8 | 1.9 |
+| `CPUManager` | `true` | Beta | 1.10 | 1.25 |
+| `CPUManager` | `true` | GA | 1.26 | 1.26 |
```

Comment:
I think we need to move this to `/docs/reference/command-line-tools/reference/feature-gates/#feature-gates-for-graduated-or-deprecated-features` as in this file above I noticed the [text](https://github.com/kubernetes/website/blame/main/content/en/docs/reference/command-line-tools-reference/feature-gates-removed.md#L9-L18) which seems to indicate that we shouldn't have feature gates corresponding to GA features here:
```

This page contains list of feature gates that have been removed. The information on this page is for reference.
A removed feature gate is different from a GA'ed or deprecated one in that a removed one is
no longer recognized as a valid feature gate.
However, a GA'ed or a deprecated feature gate is still recognized by the corresponding Kubernetes
components although they are unable to cause any behavior differences in a cluster.
For feature gates that are still recognized by the Kubernetes components, please refer to
the [Alpha/Beta feature gate table](/docs/reference/command-line-tools/reference/feature-gates/#feature-gates-for-alpha-or-beta-features)
or the [Graduated/Deprecated feature gate table](/docs/reference/command-line-tools/reference/feature-gates/#feature-gates-for-graduated-or-deprecated-features)
```

Reply:
thanks! I'll check ASAP and update as needed.